autopush/web/webpush.py

```diff
@@ -426,6 +426,10 @@ def post(self,
             version=notification.version
         )
 
+        if notification.data:
+            self.metrics.increment("updates.notification.encoding.{}".format(
+                notification.headers['encoding']))
```

Comment:
Nope, it's 'encoding' once we finish schema processing. 
https://github.com/mozilla-services/autopush/blob/master/autopush/utils.py#L352

Will add the test.